### PR TITLE
Project 1.0

### PR DESCRIPTION
**Pull request by AlexandruGH**
Greetings! I still have that problem with push .I tried several times to change the url from push option manage->git->remote but I have received git error 128 every time.I also search for a solution but didn't manage to solve this bug.
About the project,I made some modifications ,but I have a problem with the displayCode area,the area that displays the code that is executed at the specific time,because it doesn't display the modifications in real time ,but only at the end of all the process.I tried Thread.sleep() method in its printFunction code, but it just slows down all the process.I found some solutions on the web with SwingWorker class but it looks a little difficult.

### CODE REVIEW

File: MyOwnRepository/OOP-2015-master/Students/30425/Ghiurutan V. Alexandru/Project_Huffman/src/huffman/views/Gui.java [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+				if (inputText.equals("")) {
+					JOptionPane.showMessageDialog(Gui.this, "Please enter some text.", "Error",
+							JOptionPane.ERROR_MESSAGE);
+					System.exit(0);
+				}
+				inputMessageArea2.setText("");
+				outputMessageArea1.setText("");
+				outputMessageArea2.setText("");
+				huffman = new Huffman(inputText);
+				huffman.getSymbolsFrequency();
```
Comment (AlexandruGH) [inline MyOwnRepository/OOP-2015-master/Students/30425/Ghiurutan V. Alexandru/Project_Huffman/src/huffman/views/Gui.java]: Here codeDisplay doesn't do anything.

File: MyOwnRepository/OOP-2015-master/Students/30425/Ghiurutan V. Alexandru/Project_Huffman/src/huffman/views/DrawingWest.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
+package huffman.views;
+
+import com.mxgraph.swing.mxGraphComponent;
+import com.mxgraph.view.mxGraph;
+import javax.swing.JPanel;
+import huffman.models.Huffman;
+import huffman.models.Node;
+
```
Comment (AlexandruGH) [inline MyOwnRepository/OOP-2015-master/Students/30425/Ghiurutan V. Alexandru/Project_Huffman/src/huffman/views/DrawingWest.java]: Tree drawing works.

**Comment by Alex-Cosma** [comment]
Could you please put the jgraphx library in the project root folder so I can run the project as well? Thanks!

**Comment by AlexandruGH** [comment]
Sorry for that,I added the jgraphx.jar library and also the executable jar of the project .I hope it works now.

**Comment by Alex-Cosma** [comment]
You can use a [Timer](http://www.tutorialspoint.com/java/util/timer_schedule_period.htm). Will it solve your issue?

File: MyOwnRepository/OOP-2015-master/Students/30425/Ghiurutan V. Alexandru/Project_Huffman/src/huffman/views/Gui.java [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+		scroll6.setBorder(border);
+		outputAreaSouth.add(scroll5);
+		outputAreaSouth.add(scroll6);
+		this.add(outputAreaSouth, BorderLayout.SOUTH);
+	}
+
+	private class Handler implements ActionListener {
+		@Override
+		public void actionPerformed(ActionEvent event) {
+			if (event.getSource() == button1 && (clickCount == 0)) {
```
Comment (AlexandruGH) [inline MyOwnRepository/OOP-2015-master/Students/30425/Ghiurutan V. Alexandru/Project_Huffman/src/huffman/views/Gui.java]: Here I made the modifications.

**Comment by AlexandruGH** [comment]
Merry Christmas! I used a different kind of approach ,using a next button and it seems to work well now.I also update the modifications on the executable jar.I'm expecting your review.

**Comment by Alex-Cosma** [comment]
You've done a great job. I have nothing more to add.
Enjoy the rest of your holiday! :blush:

**Comment by Alex-Cosma** [comment]
Great! You're 100% done now. 
Enjoy the rest of your holiday :relaxed:

**Comment by AlexandruGH** [comment]
Thanks .I also added the UML diagram ,but I don't figure out how can I move this file in students/Ghiurtuan V. Alexandru folder.

**Comment by Alex-Cosma** [comment]
I did that for you and committed the changes locally. They will appear on the main repo when I will push everything

**Comment by AlexandruGH** [comment]
Hi! I don't know why the project doesn't appear anywhere in the repository.

**Comment by Alex-Cosma** [comment]
Weird! I do remember merging it and seeing it there. I'll double check tomorrow

**Pull request merged**